### PR TITLE
Overrides to sedml

### PR DESCRIPTION
**Pull request by danv61**

### CODE REVIEW

**Comment by moraru** [comment]
@danv need to verify syntax and execution with other simulator for test model containing mix of scanned parameters and dependent functions

File: vcell-core/src/main/java/org/vcell/sedml/SEDMLExporter.java [hunk 1]
```diff
@@ -512,6 +538,7 @@ private void translateBioModelToSedML(String savePath, String sBaseFileName, boo
 
 							for (String unscannedParamName : unscannedParamHash.values()) {
 								SymbolTableEntry ste = getSymbolTableEntryForModelEntity(mathSymbolMapping, unscannedParamName);
+								assert ste != null;
```
Comment (jcschaff) [inline vcell-core/src/main/java/org/vcell/sedml/SEDMLExporter.java:541]: throw new RuntimeException("failed to retrieve vcell parameter from unscanned SEDML parameter "+unscannedParamName+" using mathSymbolMapping);

Comment (jcschaff) [reply vcell-core/src/main/java/org/vcell/sedml/SEDMLExporter.java:541]: we should probably avoid assert statements in production code.

Comment (moraru) [reply vcell-core/src/main/java/org/vcell/sedml/SEDMLExporter.java:541]: I would clean those up once we also do proper log4j logging in SEDMLexporter... class also needs some refactoring of giant method that does the actual export

**Review by jcschaff — APPROVED** [review]
approved, waiting for integration of 3 branches to do final testing.

**Pull request merged**